src/sentry/integrations/slack/utils.py

```diff
@@ -286,3 +290,38 @@ def build_attachment(group, event=None, tags=None, identity=None, actions=None,
         'color': color,
         'actions': payload_actions,
     }
+
+
+def build_incident_attachment(incident):
```

Comment:
Can you post a screenshot of what this looks like?

Reply:
Not overly pretty at the moment. This is just the initial implementation, ideally we'd like a graph of the events in there at some point as well, and maybe a better description.
<img width="406" alt="Screenshot 2019-06-10 16 07 28" src="https://user-images.githubusercontent.com/6288560/59232627-0ebcf600-8b9a-11e9-9624-b94cde7419f7.png">